huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_consumer_group.go

```diff
@@ -0,0 +1,300 @@
+package dms
+
+import (
+	"context"
+	"regexp"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"github.com/chnsz/golangsdk"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+	"github.com/jmespath/go-jmespath"
+)
+
+func ResourceDmsRocketMQConsumerGroup() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceDmsRocketMQConsumerGroupCreate,
+		UpdateContext: resourceDmsRocketMQConsumerGroupUpdate,
+		ReadContext:   resourceDmsRocketMQConsumerGroupRead,
+		DeleteContext: resourceDmsRocketMQConsumerGroupDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"instance_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the ID of the rocketMQ instance.`,
+			},
+			"broadcast": {
```

Comment:
please put the optional argument behind of required arguments.

Reply:
done